src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -60,28 +60,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     request.getRequestDispatcher("/WEB-INF/view/login.jsp").forward(request, response);
   }
 
-  /**
-   * This function fires when a user submits the login form. It gets the username from the submitted
-   * form data, and then adds it to the session so we know the user is logged in.
-   */
+   /**
+    * This function fires when a user submits the login form. It gets the username and password from
+    * the submitted form data, checks that they're valid, and either adds the user to the session
+    * so we know the user is logged in or shows an error to the user.
+    */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    String username = request.getParameter("username");
-
-    if (!username.matches("[\\w*\\s*]*")) {
-      request.setAttribute("error", "Please enter only letters, numbers, and spaces.");
-      request.getRequestDispatcher("/WEB-INF/view/login.jsp").forward(request, response);
-      return;
-    }
+      String username = request.getParameter("username");
```

Comment:
Formatting nits: Interesting that the sample code on the sites page doesn't exactly follow the style guide! If you'd like to clean this up to match exactly, then blocks should be indented 2 spaces for each level of nesting, if's and else's have spaces (like `if (foo.check()) {`), and else's stay on the same line as the closing brace.

There's an example here: https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style